### PR TITLE
enhancements: update docs and fix bin

### PR DESCRIPTION
**Pull request by ev1stensberg**
Updates documentation to use `webpack-cli init` instead of `webpack-cli --init` and fixes the `bin` folder accordingly.

### CODE REVIEW

File: SCAFFOLDING.md [hunk 1]
```diff
@@ -2,7 +2,43 @@
...
 Setting up webpack for the first time is hard. Writing advanced configurations to optimize performance is even harder. The `init` feature is designed to support people that want to create their own configuration or initializing other projects people create.
 
-Through [yeoman](http://yeoman.io/), the `webpack-cli --init` feature allows people to create scaffolds and generate new projects quickly. An npm dependency that scaffolds a `webpack.config.js` through `webpack-cli` is what we refer to as an **addon**.
+## webpack init
+
+Through [yeoman](http://yeoman.io/), the `webpack-cli init` feature allows people to create scaffolds and generate new projects quickly. An npm dependency that scaffolds a `webpack.config.js` through `webpack-cli` is what we refer to as an **addon**.
+
+We ask several questions in the default generator to get you started.
+
+</hr>
```
Comment (bebraw) [inline SCAFFOLDING.md]: I think `---` should be the same in Markdown like this:

---

Testing.

File: SCAFFOLDING.md [hunk 2]
```diff
@@ -2,7 +2,43 @@
...
+
+1. `Will you be creating multiple bundles? (Y/n)`
+
+What we are meaning here, is if you want to provide your bundle a single or multiple [entry points](https://webpack.js.org/configuration/entry-context/#entry). If you have only one entry to your app, answer yes. If you got more modules you want to bundle, answer no.
```
Comment (bebraw) [inline SCAFFOLDING.md]: Can you push the descriptions to the tool itself? Now it's going to be painful to maintain (two places to tweak if you change a question).

File: bin/webpack.js [hunk 3]
```diff
@@ -155,10 +155,11 @@ if(argv.verbose) {
 	argv['display-cached'] = true;
 	argv['display-cached-assets'] = true;
 }
-if(argv.init) {
-	return require('../lib/initialize')(argv._);
-} else if(argv.migrate) {
-	const filePaths = argv._;
+if(argv._.includes('init')) {
+	const initPkgs = argv._.length === 1 ? [] : [argv._.pop()];
+	return require('../lib/initialize')(initPkgs);
```
Comment (bebraw) [inline bin/webpack.js]: I would add a newline before `return` to improve readability.

File: bin/webpack.js [hunk 4]
```diff
@@ -155,10 +155,11 @@ if(argv.verbose) {
...
+} else if(argv._.includes('migrate')) {
+	const filePaths = argv._.length === 1 ? [] : [argv._.pop()];
 	if (!filePaths.length) {
```
Comment (bebraw) [inline bin/webpack.js]: Same here.

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "webpack-cli",
-  "version": "1.3.0",
+  "version": "1.3.1",
```
Comment (bebraw) [inline package.json]: Maybe leave version bump for a separate commit?

File: SCAFFOLDING.md (hunk 2 above)
Comment (ev1stensberg) [inline SCAFFOLDING.md]: I think the default generator is going to be like it is now, more or less
